amundsen_application/static/js/components/common/BadgeList/index.tsx

```diff
@@ -8,14 +8,17 @@ import { getBadgeConfig } from 'config/config-utils';
 import { Badge } from 'interfaces/Tags';
 
 export interface BadgeListProps {
-  badges: Badge[];
+  badges: any[]; // TODO replace with new badges later
```

Comment:
nit: This is personal preference but can you add your github handle to this, e.g `TODO (ttannis): ...`. Same with all temporary code and files that you are adding. Since temporary code merged into master can be a bit confusing -- and in the worst case de-stabilizing if anyone else happens to be working on those files -- I think this habit helps signal who to reach out to if it causes any problems.

Additionally...it's even easier to fix the temp code later, or even have someone else without context fix it if the comment has a good explanation. Because you/they can easily search your handle to quickly find all the spots that need changing, and then just follow any directions.